### ISSUE TITLE
Start the setup process start in the language that the iPhone is set to

### ISSUE THREAD

**Issue by jchudge** [soundscape-community/soundscape#112]
Based on user feedback

The current version of Soundscape starts the setup process in English when the app is first installed. 
Then the user could change the language in the middle of the setup. 
I'd like to have the setup process start in the language that the iPhone is set to, but I feel that the language question to choose another language should not be eliminated in case the user is bilingual. 
The app should default to English only if the phone is already set to English or if the iPhone is set to a language not supported by Soundscape.

Also, the Soundscape Language screen is missing from the U.S. English version of the App Setup in the help documentation but it's there in the U.K. English version.

**Comment by ryanThomas21** [comment]
I'll look into this

**Comment by ryanThomas21** [comment]
It seems like depending on language region the user chooses the language won't update. For example if I put Spanish U.S. as my default language in the simulator, this won't be picked up and will start the app in english. However, if I use Spanish (Spain) it will change the startup language.

**Comment by RichaShah123** [comment]
I'm going to look into this!

**Comment by JJGatchalian** [comment]
Is this patch supposed to fix the case where you can use a language variant and it should work in the same language even if a different variant is the only one available? For example, if your phone is set to Spanish U.S., you'll get the Spain Spanish language because that's the only Spanish version available. Or if your phone is set to English Australia, it changes to the first English variation, most likely U.S.

**Comment by RDMurray** [comment]
exactly. At the moment it only uses the phone language if the language and region both match. It should fall back to the same language with a different region if available.

**Comment by JJGatchalian** [comment]
I think that should be fine. Thanks.
